packages/type/tests/utils.ts

```diff
@@ -125,7 +125,7 @@ export function expectEqualType(actual: any, expected: any, options: { noTypeNam
             }
         }
     } else {
-        if (expected !== actual) throw new Error(`Invalid type ${path}: ${expected} !== ${actual}`);
+        if (expected !== actual) throw new Error(`Invalid type ${path}: expected type: ${expected} !== actual type: ${actual}`);
```

Comment:
I added these as when the error was being thrown it was often unclear which type was which (specifically - because you specify the actual type first this is confusing because they're in reverse order in the message).

If you would prefer that this be done in a separate PR, I can try and back this change out and submit that additionally. Otherwise, feel free to resolve this.